tests/runtests.py

```diff
@@ -264,6 +264,8 @@ def django_tests(verbosity, interactive, failfast, keepdb, reverse,
     failures = test_runner.run_tests(
         test_labels or get_installed(),
         extra_tests=extra_tests,
+        tags=tags,
+        exclude_tags=exclude_tags
```

Comment:
please include a trailing comma so if more options are added later, we don't have to modify this line again